profiles/pantheon_decoupled_profile/pantheon_decoupled_profile.profile

```diff
@@ -31,5 +31,9 @@ function pantheon_decoupled_install_demo_content(array &$install_state) {
   // Delete the default image field from article content type.
   FieldConfig::loadByName('node', 'article', 'field_image')->delete();
 
+  // @TODO: This manual cache clearing should not be necessary,
+  // if https://www.drupal.org/project/drupal/issues/3076544 is fixed.
+    \Drupal::database()->truncate('cache_discovery')->execute();
```

Comment:
Running this locally with a clean install things are failing with:

lando drush si pantheon_decoupled_profile

 You are about to:
 * DROP all tables in your 'pantheon' database.

 Do you want to continue? (yes/no) [yes]:
 > y

 [notice] Starting Drupal installation. This takes a while.
 [notice] Performed install task: install_select_language
 [notice] Performed install task: install_select_profile
 [notice] Performed install task: install_load_profile
 [notice] Performed install task: install_verify_requirements
 [notice] Performed install task: install_verify_database_ready
 [notice] Performed install task: install_base_system
 [notice] Performed install task: install_bootstrap_full
 [notice] Performed install task: install_profile_modules
 [notice] Performed install task: install_profile_themes
 [notice] Performed install task: install_install_profile
 [notice] Performed install task: install_configure_form

In ExceptionHandler.php line 53:
                                                                                                                                         
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'pantheon.cache_discovery' doesn't exist: TRUNCATE "cache_discovery"; Array  
  (                                                                                                                                      
  )                                                                                                                                      
                                                                                                                                         

In StatementWrapper.php line 145:
                                                                                                      
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'pantheon.cache_discovery' doesn't exist

Reply:
Added a check for this but I see an issue still in which the Image is not loading unless a cache clear is done in Home page.